mapsims/cmb.py

```diff
@@ -0,0 +1,81 @@
+import healpy as hp, numpy as np
```

Comment:
Please rename this file into `cmb/__init__.py`, so we can more easily add other files later

Reply:
Done.